frontend/src/components/ArticleDetail.tsx

```
@@ -0,0 +1,112 @@
+    }
+
+    // Handle errors
+    if (error || !article) {
+        return (
+            <article className={styles.article}>
+                <Section>
+                    <p>Fehler beim Laden des Artikels.</p>
+                    <button
+                        onClick={() => window.location.reload()}
+                        style={{marginTop: '1rem', padding: '0.5rem 1rem'}}
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)